src/Backend/Modules/Extensions/Entity/Module.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Backend\Modules\Extensions\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * This is the Module Entity
+ *
+ * @author Mathias Dewelde <mathias@dewelde.be>
+ *
+ * @ORM\Entity(repositoryClass="Backend\Modules\Extensions\Entity\ModuleRepository")
+ * @ORM\HasLifecycleCallbacks
+ * @ORM\Table(name="Module")
```

Comment:
You should use the old database table name here. So nothing breaks in other modules.
After the full Doctrine switch is done, we can remove this line in all Entities.
```php
 * @ORM\Table(name="modules")
```